braket_features/Allocating_Qubits_on_QPU_Devices.ipynb

```diff
@@ -10,7 +10,7 @@
     "\n",
```

Comment:
<p>nit: # load all calibration data (not date)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>thanks</p>